samples/componenttest/config/backup-stack/README.md

```diff
@@ -0,0 +1,19 @@
+# Config test execution
+
+- Run `run_local_minio.sh` to start a local Minio.
+- Run `create_backup_secrets.sh` to create secrets for longhorn and velero.
+- Configure Minio
+  - Navigate `http://localhost:9000`
+  - Credentials `admin123:admin123`
+  - Create access keys
+    - Name: `MY-ACCESS-KEY` Secret: `MY-ACCESS-SECRET123`
+    - Name: `MY-VELERO-ACCESS-KEY` Secret: `MY-VELERO.ACCESS-SECRET123`
+  - Create buckets
+    - `longhorn`
+    - `velero`
+- Init EcoSystem
+  - `kubectl apply -f k8s_v1_blueprint_initial_system.yaml --namespace=ecosystem`
+- Apply backup stack with component configuration
+  - `kubectl apply -f k8s_v1_blueprint_configure_backup.yaml --namespace=ecosystem`
+- Create a backup
+  - `kubectl apply -f backup.yaml --namespace=ecosystem`
```

Comment:
Not really. The logging stack is a second option i think these leads to confusion because the logging stack works out of the box. Actual, the backup-stack is the only which requires this feature.

Reply:
Ah ok, then this is fine.